<launch>
  <rosparam param="thresh/">
    high: {h: 30, s: 255, v: 255}
    low: {h: 10, s: 20, v: 0}
  </rosparam>

  <node name="yellowGoal_detector_node" pkg="ball_detector" type="ball_detector_node" output="screen">
   <remap from="image" to="/cv_camera/image_raw"/>
   <remap from="ballLocation" to="yellowGoal"/>
  </node>

</launch>
